cmd/registry/cmd/get/get.go

```diff
@@ -75,6 +77,9 @@ func Command() *cobra.Command {
 				Pattern:        pattern,
 				Filter:         filter,
 			}); err != nil {
+				if status.Code(err) == codes.NotFound {
+					cmd.SilenceUsage = true
```

Comment:
We might silence usage for other cases, but this does so for specific resource names that aren't found.
```
% registry get apis/missing          
Error: rpc error: code = NotFound desc = "projects/menagerie/locations/global/apis/missing" not found in database
```

We still print usage for invalid patterns. 
```
% registry get invalid     
Error: unsupported pattern projects/menagerie/locations/global/invalid
Usage:
  registry get PATTERN [flags]

Flags:
      --filter string   filter selected resources
  -h, --help            help for get
      --nested          include nested subresources in YAML output
  -o, --output string   output type (name|yaml|contents) (default "name")

Global Flags:
  -c, --config string              name of a configuration profile or path to config file
      --registry.address string    the server and port of the Registry API (eg. localhost:8080)
      --registry.insecure          if specified, client connects via http (not https)
      --registry.location string   the API Registry location
      --registry.project string    the API Registry project
      --registry.token string      the token to use for authorization to the API Registry
----
Need more help?
https://github.com/apigee/registry/wiki
```